controlplane/internal/imageregistry/image_registry.go

```diff
@@ -41,14 +56,25 @@ func CreateImageRegistryConfigmap(imageRegistry *corev1.ConfigMap, namespace str
 		)
 	}
 
-	imageDigestMirrors, imageTagMirrors, insecureRegistries, err := getImageRegistries(registryConf)
+	mirrorConfigs := initializeMirrorConfigs()
```

Comment:
No need to initialize explicitly, we can do it inside getImageRegistry

Reply:
Good idea, modified it!